aldryn_forms/templatetags/aldryn_forms_tags.py

```diff
@@ -7,6 +7,18 @@
 register = template.Library()
 
 
+@register.assignment_tag(takes_context=True)
+def render_notification_text(context, email_notification, email_type):
+    text_context = context.get('text_context')
+
+    if not (text_context or email_notification):
+        return
+
+    render_func = 'render_%s' % email_type
```

Comment:
I'd put a comment informing about what kind of `email_type` I can use. Maybe even some sort of assertion / exception (the one that will be raised a line below about AttributeError might be misleading).